olp-cpp-sdk-core/src/cache/DiskCache.cpp

```diff
@@ -328,19 +343,25 @@ bool DiskCache::RemoveKeysWithPrefix(const std::string& prefix) {
   if (prefix.empty()) {
     for (iterator->SeekToFirst(); iterator->Valid(); iterator->Next()) {
       batch->Delete(iterator->key());
+      data_size += iterator->value().size() + iterator->key().size();
     }
   } else {
     for (iterator->Seek(prefix);
          iterator->Valid() && iterator->key().starts_with(prefix);
          iterator->Next()) {
       batch->Delete(iterator->key());
+      data_size += iterator->value().size() + iterator->key().size();
     }
   }
 
   auto result = ApplyBatch(std::move(batch));
+  if (!result.IsSuccessful()) {
+    data_size = 0u;
+  }
 
+  removed_data_size = data_size;
   return result.IsSuccessful();
 }
 
-}  // namespace cache
+}  // namespace olp
```

Comment:
please change back, it should be `namespace cache`

Reply:
Not sure how this namespace mess happen, maybe msvc did it automatically. Fixed. :-)